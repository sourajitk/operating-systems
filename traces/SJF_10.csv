1,0,1
2,0,2
3,0,3
4,6,100
5,6,1
6,6,2
7,6,3
8,112,100
9,112,1
10,112,2
11,112,3
12,112,4
13,112,0
14,112,0
15,500,1000
16,500,0
17,500,1
18,500,0
19,500,1
20,500,0
21,500,1
22,500,0
23,500,1
24,500,0
25,500,1
26,500,0
27,500,1
28,500,0
29,500,1
30,500,0
31,500,1
