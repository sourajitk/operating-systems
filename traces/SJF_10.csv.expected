1,1
2,3
3,6
4,106
5,107
6,109
7,112
8,212
9,213
10,215
11,218
12,222
13,212
14,212
15,1500
16,1500
17,1501
18,1500
19,1502
20,1500
21,1503
22,1500
23,1504
24,1500
25,1505
26,1500
27,1506
28,1500
29,1507
30,1500
31,1508
